.github/workflows/ci.yml

```diff
@@ -15,10 +15,12 @@ on:
   push:
     branches:
       - master
-      - 'release-.*'
     tags:
       - 'v[1-9][0-9]*.[0-9]+.[0-9]+'
       - 'v[1-9][0-9]*.[0-9]+.[0-9]+rc[0-9]'
+# Make sure to ignore all current release tags
```

Comment:
might as well ignore *all* release tags instead

Reply:
Do we never deploy any tags from master? I don't have a regex for all possible release tags.

Reply:
in practice i think all release tags will match the ones you have above. we might deploy from master one day soon (for nightly builds), but we can implement that later. just get this one working first.